README.md

```diff
@@ -118,47 +121,105 @@ your editor right next to your code.
 
 ### Updating the docs
 
-The `react-magma-docs` is the project for the documentation site. Any changes to the public API of an existing component or the creation of a new component should be documented here.
+The `react-magma-docs` is the project for the documentation site. Any changes to the public API of an existing component or the creation of a new component **must** be accurately captured documented here.
 
 This project uses [MDX](https://mdxjs.com/), allowing the combination of Markdown and React components.
 
-Each component has it's own `.mdx` file. If you have created a new component you will need to create a new `.mdx` file under the `pages/api` folder. To import your new component go to the `layout.js` file, import your new component, then add it to the `scope` prop array of the `LiveProvider`. Now you are able to use the component in your newly created `.mdx` file.
+Each component has its own `.mdx` file. If you have created a new component you will need to create a new `.mdx` file under the `pages/api` folder. If you have updated an existing component, you will need to review the existing docs page
+and update it as needed.
+
+#### Creating a docs page
 
-Be sure to add in the navigation details at the top of the page in the front matter.
+Add in the navigation details at the top of the page in the front matter.
 
 ```yaml
 ---
+pageTitle: '{ComponentName} API'
 title: '{ComponentName}'
-order: 2
+props:
```

Comment:
This might be sufficient, but I'll nitpick that it's only `{ComponentName}Props` by convention and some components have more than one Interface/type that we export... something like card ¯\_(ツ)_/¯

Reply:
I think this would be good enough to just show it, but we're missing a blurb to say add `<{ComponentName}Props/>` at the end of the page

Reply:
I guess we talk about this further down 😂